src/sqlite3ppext.cpp

```diff
@@ -23,6 +23,7 @@
 // THE SOFTWARE.
 
 #include "sqlite3ppext.h"
+#include <cstring>
```

Comment:
Please, put system header files before others and put a blank line between the blocks.